exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -147,6 +147,9 @@ table.sortable thead .sorting_desc { background-image: url("/static/img/black-de
             </button>
           </form>
       </p>
+
+<#include "*/runningQuery.ftl">
```

Comment:
Thanks for doing this. May be in future we can move some of the other common code in runningQuery.ftl too.